install/froxlor.sql

```diff
@@ -703,7 +703,7 @@ opcache.interned_strings_buffer'),
 	('panel', 'customer_hide_options', ''),
 	('panel', 'is_configured', '0'),
 	('panel', 'version', '0.10.10'),
-	('panel', 'db_version', '201912310');
+	('panel', 'db_version', '201912313');
```

Comment:
Missing the new setting in `install/froxlor.sql` - new installations wont be able to save the setting

Reply:
I was pretty sure I did not forget something this time. Well... Thanks for your patience with me! :)